src/applications/vaos/components/calendar/CalendarCell.jsx

```diff
@@ -121,7 +121,9 @@ const CalendarCell = ({
           renderIndicator({ date, id, selectedDates })}
         {inSelectedArray &&
           !renderIndicator && (
-            <i className="fas fa-check vads-u-color--white vaos-calendar__fa-check-position" />
+            <span className="vads-u-color--white vaos-calendar__fa-check-position">
+              <va-icon icon="check" size="3" aria-hidden="true" />
+            </span>
```

Comment:
Adding comment here since GH doesn't allow comments that aren't part of the diff. Since this file is being modified anyway, can you add the props validation so the warnings at the top of the file are resolved?

Reply:
@JunTaoLuo, Thanks for looking over this PR. PropTypes are added to the component file. Would you review this PR again?

Reply:
Nice!